tests/crypto/sse/findex/implementations/redis/tests/cloudproof.test.ts

```diff
@@ -59,7 +59,12 @@ test("upsert+search", async () => {
       LABEL,
       "france spain",
       false,
-      1000
+      1000,
+      1000,
+      (res: Uint8Array[]) => {
+        console.log(res);
```

Comment:
Maybe add a test to check if the `progress` callback receive all the element by incrementing a counter and comparing with the final result?